### PR TITLE
total ascent

### PR DESCRIPTION
**Pull request by aradmargalit**
Closes https://github.com/aradmargalit/milameter/issues/60

Turns out we have access to `messages.sessionMsgs`, which has a bunch of summary data 😎 

```ts
avgCadence: 89
avgFractionalCadence: 0.140625
avgHeartRate: 150
avgRunningCadence: 89
avgSpeed: 2.997
avgTemperature: 13
enhancedAvgSpeed: 2.997
enhancedMaxSpeed: 3.551
event: "lap"
eventType: "stop"
firstLapIndex: 0
maxCadence: 96
maxFractionalCadence: 0.5
maxHeartRate: 167
maxRunningCadence: 96
maxSpeed: 3.551
maxTemperature: 24
messageIndex: 0
necLat: 399985833
necLong: -1403263890
numLaps: 4
sport: "running"
startPositionLat: 399985833
startPositionLong: -1403286685
startTime: Wed Feb 08 2023 06:46:24 GMT-0800 (Pacific Standard Time) {}
subSport: "generic"
swcLat: 399828124
swcLong: -1403396067
timestamp: Wed Feb 08 2023 07:19:39 GMT-0800 (Pacific Standard Time) {}
totalAnaerobicTrainingEffect: 0
totalAscent: 95
totalCalories: 331
totalCycles: 2486
totalDescent: 98
totalDistance: 5038.62
totalElapsedTime: 1681.009
totalStrides: 2486
totalTimerTime: 1681.009
totalTrainingEffect: 3.2
trigger: "activityEnd"
```

image

### CODE REVIEW

**Comment by aradmargalit** [comment]
@eshedmargalit  as we add more of these, we may want to consider a table view, like


| human | dog | difference | %diff |
| --- | --- | --- | --- |
| 1 mi | 1.5 mi | +0.5mi | +50% |
| 8:07 min/mi | 7:02 min/mi | -1:05 min/mi | -20% |

The initial build is complicated, but adding new statistics would be trivially easy.

The downsides:
* takes up a lot of space (maybe we move this all below the map?)
* adding additional dogs in future may be harder

**Comment by eshedmargalit** [comment]
I think that if we can't really imagine more than 1-2 future statistics of interest, better to keep the implementation as-is. The table view is a little hard to parse and visually doesn't look as good imo

But if there were like 10 stats of interest, I'd be in favor

**Review by eshedmargalit — APPROVED** [review]
Looks great
